.github/workflows/ci.yaml

```
@@ -17,35 +17,38 @@ on:
           - db: pgsql
             database_url: postgres://postgres:password@127.0.0.1:5432/eccube_db
             database_server_version: 14
+            database_charset: utf8
     services:
-      mysql:
-        image: mysql:5.7
+      mysql8:
+        image: mysql:8
         env:
           MYSQL_ROOT_PASSWORD: password
+          MYSQL_DATABASE: ${{ matrix.dbname }}
```

actionlint (1.7.12)

[error] property "dbname" is not defined in object type {database_charset: string; database_server_version: number; database_url: string; db: string; eccube_version: number; php: number; plugin_code: string}

(expression)